README.md

```diff
@@ -1,3 +1,25 @@
 # HTML-project
 
 Hey there, welcome to our HTML open source project. Redesign the HTML landing and have fun doing it. Happy coding :)
+
+# Contribution Guide
+
+- Fork the repository, clone and make sure you are not working on the `master branch`
+
+```bash
+git clone https://github.com/joshuafolorunsho/hmtl-project.git
```

Comment:
just a suggestion, instead of typing it here at `line 10` as: 
`https://github.com/joshuafolorunsho/hmtl-project.git `

because others might make a mistake of your forked repo as theirs while cloning, i guess it would be a better way as `https://github.com/{your-github-username}/hmtl-project.git`

or briefly explain that part as a side note 😃

Reply:
Thanks Itshally, I'll fix it right away

Reply:
What do you think right now? @itshally

Reply:
it looks great 👍

Reply:
Thanks :)